src/test/java/net/mcreator/integration/TestWorkspaceDataProvider.java

```diff
@@ -248,15 +248,28 @@ private static GeneratableElement getExampleFor(ModElement modElement, Random ra
 					ListUtils.getRandomItem(random, ElementUtil.loadBlocks(modElement.getWorkspace())).getName());
 			biome.undergroundBlock = new MItemBlock(modElement.getWorkspace(),
 					ListUtils.getRandomItem(random, ElementUtil.loadBlocks(modElement.getWorkspace())).getName());
-			biome.vanillaTreeType = "Mega spruce trees";
 			biome.airColor = Color.red;
 			if (!emptyLists) {
 				biome.grassColor = Color.green;
 				biome.foliageColor = Color.magenta;
 				biome.waterColor = Color.blue;
 				biome.waterFogColor = Color.cyan;
 			}
-			biome.treesPerChunk = new int[] { 0, 5, 10, 16 }[valueIndex];
+			if(biome.customTree) {

```

Comment:
You check biome.customTree, but never set biome.customTree in tests so this will always be false and custom trees won't even be tested by the unit tests.